gem/make/pe.mk

```diff
@@ -0,0 +1,5 @@
+SELF_DIR := $(dir $(lastword $(MAKEFILE_LIST)))
+include $(SELF_DIR)/common.mk
+
+NAMESPACE ?= artifactory.delivery.puppetlabs.net/pe-and-platform
+LATEST_VERSION ?= kearney-latest
```

Comment:
NOTE this is some PE-related stuff that is in a public repo.

Maybe that's okay considering the content, but something we should think about.

Reply:
Oh I guess we PE-specific details like this in here already....so nothing new about this stuff being added.